include/mbedtls/sha512.h

```diff
@@ -263,13 +269,14 @@ int mbedtls_sha512_ret( const unsigned char *input,
  *
  * \deprecated     Superseded by mbedtls_sha512_ret() in 2.7.0
  *
- * \param input    The buffer holding the data.
- *                 Must not be \c NULL if \p ilen is greater than 0.
- * \param ilen     The length of the input data.
- * \param output   The SHA-384 or SHA-512 checksum result.
- *                 Must not be \c NULL.
- * \param is384    Determines which function to use:
- *                 0: Use SHA-512, or 1: Use SHA-384.
+ * \param input    The buffer holding the data. This must be a
+ *                 readable buffer of length \p ilen Bytes. It
+ *                 must not be \c NULL.
+ * \param ilen     The length of the input data \p input in Bytes.
+ * \param output   The SHA-384 or SHA-512 checksum result. This must
+ *                 be a writable buffer of length \c 64 Bytes.
+ * \param is384    Determines which function to use. This must be eiher
```

Comment:
Typo: eiher -> either

Reply:
Thanks @dgreen-arm, I will fix this.

Please note that the question of whether we want to accept `NULL` input data is still being discussed, so this part of the PR might be reworked.